python/paddle/vision/ops.py

```diff
@@ -1319,8 +1322,8 @@ def read_file(filename, name=None):
             [142773]
     """
 
-    if in_dygraph_mode():
-        return _legacy_C_ops.read_file('filename', filename)
+    if in_dynamic_or_pir_mode():
+        return _C_ops.read_file(filename)
```

Comment:
从 ci 报错来看，这里应该还不能统一起来写，还是分开吧~
```suggestion
    if in_dygraph_mode():
        return _legacy_C_ops.read_file('filename', filename)
    elif in_pir_mode():
        return _C_ops.read_file(filename)
```

Reply:
收到